server/routes/conditions.ts

```diff
@@ -35,6 +36,41 @@ export default ({ licenceService, conditionsService }: { licenceService: Licence
       })
     }
 
+    router.post('/standard/:bookingId', audited, asyncMiddleware(postStandard))
```

Comment:
these were handled by the standard router but now we need additional bespoke code